testdata/p4_16_samples/factory1.p4

```diff
@@ -13,9 +13,16 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+#include <core.p4>
```

Comment:
I don't think you should delete the original file factory1.p4.

Reply:
I see, git things you actually renamed issue345, but in fact you have deleted it.
Nevermind this comment.